src/som/interpreter/actors/EventualMessage.java

```diff
@@ -20,6 +21,7 @@
   protected final Object[]  args;
   protected final SResolver resolver;
   protected final RootCallTarget onReceive;
+  protected final EventualMessage causalMessage;
   protected final long causalMessageId;
```

Comment:
do we really need two fields here?
I suppose `causalMessageId` is part of the `causalMessage`?